Trisha-tech/OnlineBookSales
# Shop products page is added - issue #266

### Description

- A responsive layout that displays a list of available products with images, descriptions, and prices.
- Functional Search buttons, ensuring a smooth user experience.
- A navigation menu for easy access to other sections of the website, including the home page and cart.
- Adherence to the website's design guidelines for consistency in styling and user interface.

### Related Issues
- Placeholder: "Fixes #266"

### Changes
- Fixed a bug in Shop page is not available / gives 404 error

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Run `npm install` to install dependencies.
3. Run `npm test` to execute the test suite.
4. Verify that the shop page is visible

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/5ef5d4fd-ad38-40fb-96a6-6282d3486a8e)
